src/data/contributors/ sonuKushwaha.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
@NikkiPriya 
You have mistakenly deleted the data of Sonu Kushwaha. Please restore it.